packages/protvista-variation-adapter/src/protvista-variation-adapter.ts

```diff
@@ -13,10 +20,32 @@ const getSourceType = (xrefs: Xref[], sourceType: SourceType) => {
   return xrefNames;
 };
 
-export const transformData = (data: ProteinsAPIVariation) => {
+export type TransformedVariant = {
```

Comment:
If I got this correctly, this is the data in the shape that is required by protvista-variation, right?
Maybe this should actually go into this package and get in imported here for this function's output.